### PR TITLE
Update zh-cn (i18n)

### PR DESCRIPTION
**Pull request by Felix2yu**
php cli/check.translation.php -d -l=zh-cn -r
Language: zh-cn

zh-cn - Translation is 100.0% complete.

### CODE REVIEW

File: app/i18n/zh-cn/install.php [hunk 1]
```diff
@@ -33,89 +33,89 @@
...
 		'xml' => array(
-			'nok' => '找不到用于 XML 解析库。',
-			'ok' => '已找到 XML 解析库。',
+			'nok' => '找不到用于 XML 解析库',
+			'ok' => '已找到 XML 解析库',
 		),
 		'_' => '检查',
 	),
 	'conf' => array(
-		'ok' => '常规配置已保存。',
```
Comment (Frenzie) [inline app/i18n/zh-cn/install.php:101]: Just curious, is that `。` just a regular `.` (from our Latin perspective)?

**Review by Frenzie — APPROVED** [review]
Don't read Chinese, but looks fine. ;-)

File: app/i18n/zh-cn/install.php (hunk 1 above)
Comment (9uanhuo) [reply app/i18n/zh-cn/install.php:101]: Yes, that mean the end of a sentence. So it is used just like the ' . ' in Latin in most time.

**Comment by Alkarex** [comment]
Thanks @Felix2yu !
Could you please add a line for you in https://github.com/FreshRSS/FreshRSS/blob/master/CREDITS.md ?

**Pull request merged**